svthalia/Reaxit
# Feature/settings screen

Closes #23 

Added FireBase messaging. The App will now automatically register devices for push notifications. Changes to the settings screen so that users can change their notification preferences.

I have not added push notification routing yet, there are TODO's for that in the code. We need to port deepLinking.js from the old app if we want routing.

How to test:
1. Check the settings screen for the available notification settings.
2. Check the code for the setup of FireBaseMessaging
